### PR TITLE
Migrate from Travis CI to GitHub actions

### LINKED ISSUE #165: Set up GitHub actions for CI
Maybe we can toss out our Travis CI configuration and replace it with GitHub actions.

We tried migrating from Travis to Circle before, but couldn't dynamically start Docker containers without hassle.
It looks like [mint](https://github.com/elixir-mint/mint) for instance, also uses Docker with GitHub actions:
https://github.com/elixir-mint/mint/blob/v1.2.0/.github/workflows/main.yml

So maybe that's a way forward. 🧑‍🔬

### PR DESCRIPTION
**Pull request by pmeinhardt**
Closes #165

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+          elixir-version: "1.11.2"
+
+      - name: Install dependencies
+        run: mix deps.get
+
+      # TODO: Enable in next release
+      #
+      # At the moment, we still have pending PRs etc.
+      # and introducing auto-formatting now would cause
+      # a ton of conflicts.
```
Comment (tessi) [inline .github/workflows/ci.yml:27]: ❤️

**Review by tessi — APPROVED** [review]
looks really, really cool

File: .github/workflows/ci.yml (hunk 1 above)
Comment (klappradla) [inline .github/workflows/ci.yml:27]: 💚 that you already prepared for this 💚

**Review by klappradla — COMMENTED** [review]
Love your efforts of keeping us connected to the future 🐎

**Pull request merged**